INA219.cpp

```diff
@@ -77,16 +77,11 @@ void INA219::begin() {
 
 void INA219::calibrate(float shunt_val, float v_shunt_max, float v_bus_max, float i_max_expected) {
     uint16_t digits;
-    float i_max_possible, min_lsb, max_lsb, swap;
+    float min_lsb, swap;
 
-#if (INA219_DEBUG == 1)
```

Comment:
Could you put all the variables definitions here ? It looks more like classic C style.